ansible_collections/arista/avd/molecule/eos_validate_state/prepare.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Prepare
+  hosts: FABRIC
+  gather_facts: false
+  vars:
+    root_dir: '{{ playbook_dir }}'
+  tasks:
+    - name: Generate AVD Structured Configurations and Fabric Documentation
+      ansible.builtin.import_role:
+        name: arista.avd.eos_designs
+      vars:
+        avd_data_validation_mode: error
+        avd_data_conversion_mode: error
+      # This is terrible but that's the best I can do with molecule and our
+      # roles disjointed tags
+      tags: always
```

Comment:
Are you sure this is actually doing what you want? Notice the difference between include_role and import_role in regards to tags behavior.

Reply:
so basically legacy `eos_validate_state` is .

If I have a playbook and I run `eos_designs` AND `eos_validate_state` and I give the tag `build` to `ansible-playbook` we will skip all the tasks in `eos_validate_state` 

So here what I am doing in molecule (molecule does NOT allow to give different tags to different steps sadly)
is that I am making all the `eos_designs` task with always and then I am running with `skip-tags=documentation,debug`

so this does what I want but it makes me sad

Reply:
As agreed offline we will change to have an input variable for enabling anta mode instead.

Reply:
Base on our chat. we will drop the `anta` tags and instead use a variable to activate `anta` from the group_vars. Will work on it.

It will keep everybody saner and I will be able to remove my sad hack.

Plus this sad hack implies that we CANNOT use `eos_designs` in the same playbook as `eos_validate_state with `--tags=anta` as otherwise no step from `eos_designs` would be run. Which is bad and will make users sad.

Reply:
Addressed in [c6d9d58](https://github.com/aristanetworks/ansible-avd/pull/3171/commits/c6d9d585cae5d24e71ea981859c3321165a4d515)